rules-emerging-threats/2025/Exploits/CVE-2025-29824/file_event_win_exploit_cve_2025_29824.yml

```diff
@@ -0,0 +1,31 @@
+title: Potential Exploitation of CVE-2025-29824 - CLFS BLF File Creation
+id: 4a8e9f3d-7b2c-4f1a-9e5c-8d3b7f5c1234
+status: experimental
+description: |
+    Detects the suspicious creation of CLFS BLF files which may indicate possible exploitation of CVE-2025-29824.
+    CVE-2025-29824 is a elevation of privilege vulnerability in the Windows Common Log File System (CLFS) driver.
+    According to Microsoft, this vulnerability is being exploited in the wild by Storm-2460, a ransomware group to
+    gain elevated privileges to deploy ransomware.
+references:
+    - https://www.microsoft.com/en-us/security/blog/2025/04/08/exploitation-of-clfs-zero-day-leads-to-ransomware-activity/
+author: Swachchhanda Shrawan Poudel (Nextron Systems)
+date: 2025-04-10
+tags:
+    - attack.privilege-escalation
+    - attack.t1068
+    - cve.2025-29824
+    - detection.emerging-threats
+logsource:
+    category: file_event
+    product: windows
+detection:
+    selection_blf:
+        TargetFilename|endswith: '.blf'
+    selection_path:
+        TargetFilename|contains: ':\ProgramData\SkyPDF\'
+    selection_image:
+        Image|endswith: '\dllhost.exe'
```

Comment:
Who is the process that usually writes `.blf` files? Are you sure its not dllhost?